src/components/MemoDetail.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="backdrop" @click.self='closeDetail'>
+    <div class="modal">
+      <h2>Memo Detail</h2>
+      <label>
+        <input placeholder="Title" v-model='this.memo.title'>
+      </label>
+      <label>
+        <textarea placeholder="Content" v-model='this.memo.content'></textarea>
```

Comment:
`this` はなくても動いたりしませんかね...？

```suggestion
        <textarea placeholder="Content" v-model='memo.content'></textarea>
```